src/test/java/fi/fmi/avi/util/GTSExchangeFileInfoTest.java

```diff
@@ -31,6 +27,7 @@ public void testFileNameGenerator() {
                         .setDataTypeDesignatorT1ForTAC(DataTypeDesignatorT1.FORECASTS)
                         .setDataTypeDesignatorT2(DataTypeDesignatorT2.ForecastsDataTypeDesignatorT2.FCT_AERODROME_VT_LONG)//
                         .setType(BulletinHeading.Type.CORRECTED)//
+                        .setBulletinAugmentationIndicator("CCA")
```

Comment:
Hmm, never thought of this from perspective of populating the `BulletinHeading` object before. I see the typical case of setting the `bulletinAugmentationIndicator()` seems redundant and error-prone. Maybe the indicator string could be set by constructing it somehow from type and bulletin augmentation number. Some examples of implementation alternatives to consider:

* `static String BulletinHeading.bulletinAugmentationIndicator(Type, int)` - a simple static utility method, maybe the safest bet
* `Builder BulletinHeadingImpl.Builder.setBulletinAugmentationIndicator(Type, int)`
    * Would this set only the String value or type and number as well? Probably then should have a different name than the String setter to avoid confusion.
    * This could be implemented internally invoking the static utility method above.
* `Builder BulletinHeadingImpl.Builder.setBulletinAugmentationIndicator()` setting value from already set type and number
    * slightly error-prone, as depends on invocation order
* or something completely different, or a variation of the examples above, or ...

Reply:
I opted for the second approach. Though I'm not sure how good of a solution it is now that it calls `BulletinHeadingEncoder` when setting the "original" augmentation indicator string.

Reply:
Good, probably the optimal solution here: safe but convenient. I had completely forgotten about existence of `BulletinHeadingEncoder.encodeBBBIndicator(BulletinHeading.Type, int)`.